site/docs/roadmap-overview.md

```diff
@@ -0,0 +1,57 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Roadmap Overview
+
+This is an overview of the current Roadmap planning with links to their plannings boards that encapsulate related issues.
+The mailing list discussion around Roadmap planning can be found [here](https://lists.apache.org/thread.html/r84e80216c259c81f824c6971504c321cd8c785774c489d52d4fc123f%40%3Cdev.iceberg.apache.org%3E).
+Related design docs (if available) should be linked on the planning boards.
+
+# Priority 1
+
+* API: [Iceberg 1.0](https://github.com/apache/iceberg/projects/3) [medium]
```

Comment:
@rdblue are we planning to make the next release 1.0.0? If so, I think we can remove this

Reply:
I think we need to do a review of the APIs, add documentation about guarantees, and start checking binary compatibility if we want to make the next release 1.0. I think it's a good idea to track it here.